src/Sylius/Behat/Context/Api/Admin/ManagingProductReviewsContext.php

```diff
@@ -0,0 +1,237 @@
+    {
+        Assert::true(
+            $this->isItemOnIndex('title', $title),
+            sprintf('Product review with title %s does not exist', $title)
+        );
+    }
+
+    /**
+     * @Then I should see a single product review in the list
+     * @Then I should see :amount reviews in the list
+     */
+    public function iShouldSeeReviewsInTheList(int $amount = 1): void
+    {
+        Assert::same($this->client->countCollectionItems(), $amount);
+    }
+
+    /**
+     * @When I want to modify the :productReview product review
+     */
+    public function iWantToModifyTheProductReview(ReviewInterface $productReview): void
+    {
+        $this->client->buildUpdateRequest('product_reviews', (string) $productReview->getId());
+    }
+
+    /**
+     * @When I change its title to :title
+     */
+    public function iChangeItsTitleTo(string $title): void
+    {
+        $this->client->addRequestData('title', $title);
+    }
+
+    /**
+     * @When I change its comment to :comment
+     */
+    public function iChangeItsCommentTo(string $comment): void
+    {
+        $this->client->addRequestData('comment', $comment);
+    }
+
+    /**
+     * @When /^I remove its (comment|title)$/
+     */
+    public function iRemoveItsComment(string $element): void
+    {
+        $this->client->addRequestData($element, '');
+    }
+
+    /**
+     * @When I save my changes
+     * @When I try to save my changes
+     */
+    public function iSaveMyChanges(): void
+    {
+        $this->client->update();
+    }
+
+    /**
+     * @Then /^(this product review) (comment|title) should be "([^"]+)"$/
+     */
+    public function thisProductReviewElementShouldBeValue(ReviewInterface $productReview, string $element, string $value): void
+    {
+        $this->assertIfReviewHasElementWithValue($productReview, $element, $value);
+    }
+
+    /**
+     * @Then /^(this product review) rating should be (\d+)$/
+     */
+    public function thisProductReviewRatingShouldBe(ReviewInterface $productReview, int $rating): void
+    {
+        $this->assertIfReviewHasElementWithValue($productReview, 'rating', $rating);
+    }
+
+    /**
+     * @When I choose :rating as its rating
+     */
+    public function iChooseAsItsRating(int $rating): void
+    {
+        $this->client->addRequestData('rating', $rating);
+    }
+
+    /**
+     * @When I accept the :productReview product review
+     */
+    public function iAcceptTheProductReview(ReviewInterface $productReview): void
+    {
+        $this->client->applyTransition('product_reviews',(string) $productReview->getId(), ProductReviewTransitions::TRANSITION_ACCEPT);
+    }
+
+    /**
+     * @When I reject the :productReview product review
+     */
+    public function iRejectTheProductReview(ReviewInterface $productReview): void
+    {
+        $this->client->applyTransition('product_reviews', (string) $productReview->getId(), ProductReviewTransitions::TRANSITION_REJECT);
```

Comment:
What would you say to merge this step the previous one - `I (accept|reject) the :productReview review`? I know we're then not using consts, but IMO we can deal with this danger, as it's a Behat code and not extended by anyone 💃

Reply:
I will fix it in next PR